packages/selection/src/SelectionContainer.spec.tsx

```diff
@@ -107,317 +106,322 @@ describe('SelectionContainer', () => {
     });
 
     it('first item in container defaults as the only initial focusable item', () => {
-      const { getAllByTestId } = render(<BasicExample />);
-      const [item] = getAllByTestId('item');
+      const { getByText } = render(<BasicExample />);
+      const item = getByText('Item-1');
+
+      expect(document.activeElement).not.toBe(item);
+
+      userEvent.tab();
 
-      fireEvent.focus(item);
```

Comment:
I recently learned that the Testing Library author and contributors recommend against using `fireEvent.focus` because although it triggers focus events, _it does not set the focused element as the document's `activeElement`_. So in this old test, `document.activeElement === document.body` is `true` after `fireEvent.focus(trigger)`.

Before `user-event` library existed, the community recommended using `el.focus()` instead. Now that `user-event` library can be used, the most appropriate user event should be used to simulate a focus action.

https://testing-library.com/docs/guide-events#focusblur

https://github.com/testing-library/user-event/issues/350#issuecomment-505997604